web/styles/compose.css

```diff
@@ -76,6 +76,7 @@
 
 .autocomplete_secondary {
     opacity: 0.8;
+    font-size: 85%;
```

Comment:
Similarly, if the `small` class doesn't do anything now, we can remove it from the HTML to help simplify that.

Reply:
We do not use `small` class but `<small>` element.